features/support/cucumber_table_builder.rb

```diff
@@ -0,0 +1,59 @@
+# Makes it easy to build DRY Cucumber-compatible tables
+#
+# Example
+#
+#   table_builder = CucumberTableBuilder.new ['AUTHOR', 'MESSAGE', 'FILES']
+#   table_builder.add ['Jay', 'hello', 'jay.txt']
+#   table_builder.add ['Jane', 'hi', 'jane.txt']
+#   table_builder.to_table
+#
+#   =>  [ ['AUTHOR', 'MESSAGE', 'FILES'],
+#         ['Jay', 'hello', 'jay.txt'],
+#         ['Jane', 'hi', 'jane.txt'] ]
+class CucumberTableBuilder
+
+  attr_reader :table
+
+
+  def initialize headers
+    @headers = headers
+
+    # The resulting Cucumber-compatible table structure
+    @table = [headers]
+
+    # The previously added row of values
+    @previous_values = nil
+  end
+
+
+  # Adds the given row of values to the table
+  def add_row values
+    @table << dry_up(values)
+    @previous_values = values
+  end
+
+
+  # Returns a dried up version of the given values
+  # based on the row that came before in the table
```

Comment:
Please explain in the comment what a "dried up version" of the table is
Something along the line:
```
From left to right, remove any values that match the previous row, stopping on the first difference
```